bluebrain/repo-patches/packages/steps/package.py

```diff
@@ -46,7 +46,7 @@ class Steps(CMakePackage):
     depends_on("petsc~debug+int64+mpi", when="+petsc+mpi")
     depends_on("petsc~debug+int64~mpi", when="+petsc~mpi")
     depends_on("pkg-config", type="build")
-    depends_on("py-clang-format", type=("build", "test"), when="+codechecks")
```

Comment:
Do we need this package then, still? Could be removed…

Reply:
yeah, I had to mention this `py-clang-format` package to learn about `clang-tools` from Olli ^^
Let me remove it in this PR